src/pages/monitoring/overview/index.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useEffect } from "react";
+import { MDBCard, MDBCardBody } from "mdb-react-ui-kit";
+import "./style.css";
+import { useStore } from "../../../store";
+
+type Theme = "light" | "dark";
```

Comment:
along with this type, can you plz move all the types to the type folder?

Reply:
Can you please give more details about this?